gamedb.yaml

```diff
@@ -66,6 +78,20 @@
   patches:
     - exec: rm -f installscript.vdf
 
+- name: Borderlands 2
+  platform: epic-store
+  id: Dodo
+  banner: steam:49520
+  launch_options: -nolauncher  
```

Comment:
does this actually work? I wouldn't expect it to.

Reply:
I checked several times, two games were launched without a launcher, I'm sure that this command is working.